hpy/debug/src/debug_ctx_cpython.c

```diff
@@ -97,7 +97,7 @@ void debug_ctx_CallRealFunctionFromTrampoline(HPyContext dctx,
         _HPyFunc_args_VARARGS *a = (_HPyFunc_args_VARARGS*)args;
         DHPy dh_self = _py2dh(dctx, a->self);
         Py_ssize_t nargs = PyTuple_GET_SIZE(a->args);
-        DHPy dh_args[nargs * sizeof(DHPy)]; // VLA, should be killed by #157
+        DHPy *dh_args = (DHPy *)alloca(nargs * sizeof(DHPy));
```

Comment:
I think this was wrong in #76 - it allocated too many values?

Reply:
I think you are right, thanks for fixing it